src/EditorFeatures/Core.Wpf/QuickInfo/OnTheFlyDocsViewFactory.cs

```diff
@@ -60,6 +60,8 @@ public OnTheFlyDocsViewFactory(IViewElementFactoryService factoryService, IAsync
             throw new InvalidOperationException("QuickInfoSession is null");
         }
 
+        OnTheFlyDocsLogger.LogShowedOnTheFlyDocsLink();
```

Comment:
Could we log whether a symbol has doc-comment here? That would provide us the numbers regardless of whether user clicked the link or not, right?